setup.py

```diff
@@ -35,8 +35,8 @@
           url=URL,
           version=VERSION,
           download_url=DOWNLOAD_URL,
-          long_description=open('README.rst').read(),
-          long_description_content_type='text/x-rst',
+          long_description=open('README.md').read(),
+          long_description_content_type='text/markdown',
```

Comment:
I don't understand why this was changed but whatever works

Reply:
@hoechenberger's argument was that markdown is easier for him.

That's okay for me as long as we carefully check how this renders on https://pypi.org/project/mne-bids/ before releasing (by making use of test.pypi).

Reply:
one counterargument is that rst allows you to build in sphinx-gallery. Actually, I am pretty sure that was the reason I kept it in rst initially

Reply:
> @hoechenberger's argument was that markdown is easier for him.

hum... so next time I touch I switch bck to rst cause it's easier for me :)

Reply:
> one counterargument is that rst allows you to build in sphinx-gallery without duplicating. Actually, I am pretty sure that was the reason I kept it in rst initially

yes, that was the reason (which is still valid).

Reply:
Oh my what a discussion!

So initially, `README.rst` would be what users would see when visiting the GitHub repo, and the exact same file would be included as the landing page of our documentation by Sphinx.

I totally reworked this, essentially moving the previous `README.rst` to `doc/index.rst`, and changing some things; and basically creating a new (and massively reduced) `README.md` based on a few blocks of existing text, but otherwise basically from scratch. So to me it was like creating a new file, not changing an old one, because the old one was moved.

If people feel so strongly about using reST instead of markdown, I can of course convert it to reST again

Reply:
So to summarize:

`README.md` is not included in the built docs, it's really just for those who come and stop by to look at the GH repo. The old (and updated) `README.rst` is now `doc/index.rst`.

Reply:
do not revert now that it's done. All ok on my side

Reply:
also ok from my side, I actually like a short and to-the-point GitHub README and a longer and explicit documentation index.rst

Reply:
yeah it's fine for me too :)